### ISSUE TITLE
ArgumentCountError when updating

### ISSUE THREAD

**Issue by canoine** [firefly-iii/firefly-iii#7678]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Updating from 6.0.13 to 6.0.14 causes this error :

```
(...)
      Going to verify PostgreSQL table sequences.
      Checking the next id sequence for table "2fa_tokens".

   ArgumentCountError 

  Too few arguments to function FireflyIII\Console\Commands\Upgrade\FixPostgresSequences::friendlyInfo(), 0 passed in /var/local/firefly/src/v6.0.14/app/Console/Commands/Upgrade/FixPostgresSequences.php on line 127 and exactly 1 expected

  at app/Console/Commands/ShowsFriendlyMessages.php:44
     40▕     /**
     41▕      * @param  string  $message
     42▕      * @return void
     43▕      */
  ➜  44▕     public function friendlyInfo(string $message): void
     45▕     {
     46▕         $this->friendlyNeutral($message);
     47▕     }
     48▕ 

  1   app/Console/Commands/Upgrade/FixPostgresSequences.php:127
      FireflyIII\Console\Commands\Upgrade\FixPostgresSequences::friendlyInfo()
      +10 vendor frames 

  12  app/Console/Commands/Upgrade/UpgradeDatabase.php:95
      Illuminate\Console\Command::call()
```

### Debug information

Debug information generated at 2023-06-21 08:54:56 Europe/Paris for Firefly III version **6.0.13**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.13 |
| Firefly III API | 2.0.4 |
| DB version | 19 (exp. 19) |
| Docker | No |
| PHP | `8.2.5` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Europe/Paris |
| Browser TZ | Europe/Paris |
| App environment | local |
| App debug mode | false |
| App cache driver | redis |
| App logging | notice, syslog |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 8388608 (8 MB) |
| Interface | fpm-fcgi |
| Default language | fr_FR |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | pgsql, sqlite |
| Current driver | pgsql |
| Trusted proxies (.env) | `**` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2023-01-01 00:00:00 |
| Session end | 2023-07-01 00:00:00 |
| Session first | 2020-07-31 00:00:00 |
| User ID | 2 |
| User language | fr_FR |
| User locale | fr_FR |
| Attempt at "fr_FR.utf8" | 'fr_FR.utf8' |
| Attempt at "fr_FR.UTF-8" | 'fr_FR.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/114.0 |

### Expected behaviour

Firefly should update successfully.

### Steps to reproduce

composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist /<path>/v6.0.14 v6.0.14

**Comment by JC5** [comment]
Nice find. That should be fixed in v6.0.15. If nothing else shows up today, I should be able to release it at the end of the day.

**Comment by canoine** [comment]
Better now. Thank you. :smile:
